src/mozillavpn.cpp

```diff
@@ -1381,6 +1381,11 @@ void MozillaVPN::registerUrlOpenerLabels() {
   uo->registerUrlLabel("upgradeToAnnualUrl", []() -> QString {
     return Constants::upgradeToAnnualUrl();
   });
+
+  uo->registerUrlLabel("sumoExcludedApps", []() -> QString {
+    return "https://support.mozilla.org/en-US/kb/"
```

Comment:
let's drop the "en-US" here because it stipulates that the page is shown in English. Without it, moz.org will route the page based on the person's browser language prefs.